routing/geometry.cpp

```diff
@@ -0,0 +1,122 @@
+#include "geometry.hpp"
+
+#include "geometry/mercator.hpp"
+
+#include "base/assert.hpp"
+
+namespace
+{
+using namespace routing;
+
+double constexpr kMPH2MPS = 1000.0 / (60 * 60);
+
+inline double TimeBetweenSec(m2::PointD const & from, m2::PointD const & to, double speedMPS)
+{
+  ASSERT_GREATER(speedMPS, 0.0, ());
```

Comment:
I insist that altitudes should be exposed to this formula right now - even if all altitudes are the same and set to default value.